src/test/java/com/google/jenkins/plugins/computeengine/InstanceConfigurationTest.java

```diff
@@ -184,68 +187,75 @@ public void testConfigRoundtrip() throws Exception {
-    assert (i.getMinCpuPlatform().equals(MIN_CPU_PLATFORM));
+    assertTrue(i.getName().startsWith(NAME_PREFIX));
+    assertTrue(i.getDescription().equals(CONFIG_DESC));
+    assertTrue(i.getZone().equals(ZONE));
+    assertTrue(i.getMachineType().equals(MACHINE_TYPE));
+    assertTrue(i.getMinCpuPlatform().equals(MIN_CPU_PLATFORM));
 
     // Accelerators
-    assert (i.getGuestAccelerators().get(0).getAcceleratorType().equals(ACCELERATOR_NAME));
-    assert (i.getGuestAccelerators()
-        .get(0)
-        .getAcceleratorCount()
-        .equals(Integer.parseInt(ACCELERATOR_COUNT)));
+    assertTrue(i.getGuestAccelerators().get(0).getAcceleratorType().equals(ACCELERATOR_NAME));
+    assertTrue(
+        i.getGuestAccelerators()
+            .get(0)
+            .getAcceleratorCount()
+            .equals(Integer.parseInt(ACCELERATOR_COUNT)));
 
     // Metadata
-    assert (i.getMetadata()
-        .getItems()
-        .get(0)
-        .getKey()
-        .equals(InstanceConfiguration.METADATA_LINUX_STARTUP_SCRIPT_KEY));
-    assert (i.getMetadata().getItems().get(0).getValue().equals(STARTUP_SCRIPT));
+    Optional<String> startupScript =
+        i.getMetadata().getItems().stream()
+            .filter(
+                item ->
+                    item.getKey().equals(InstanceConfiguration.METADATA_LINUX_STARTUP_SCRIPT_KEY))
+            .map(item -> item.getValue())
+            .findFirst();
+    assertTrue(startupScript.isPresent());
+    assertEquals(startupScript.get(), STARTUP_SCRIPT);
+
+    Optional<String> sshKey =
+        i.getMetadata().getItems().stream()
+            .filter(item -> item.getKey().equals(InstanceConfiguration.SSH_METADATA_KEY))
+            .map(item -> item.getValue())
+            .findFirst();
+    assertTrue(sshKey.isPresent());
+    assertFalse(sshKey.get().isEmpty());
 
     // Network
-    assert (i.getNetworkInterfaces().get(0).getSubnetwork().equals(SUBNETWORK_NAME));
-    assert (i.getNetworkInterfaces()
-        .get(0)
-        .getAccessConfigs()
-        .get(0)
-        .getType()
-        .equals("ONE_TO_ONE_NAT"));
-    assert (i.getNetworkInterfaces()
-        .get(0)
-        .getAccessConfigs()
-        .get(0)
-        .getName()
-        .equals("External NAT"));
+    assertTrue(i.getNetworkInterfaces().get(0).getSubnetwork().equals(SUBNETWORK_NAME));
+    assertTrue(
+        i.getNetworkInterfaces()
+            .get(0)
+            .getAccessConfigs()
+            .get(0)
+            .getType()
+            .equals("ONE_TO_ONE_NAT"));
+    assertTrue(
+        i.getNetworkInterfaces().get(0).getAccessConfigs().get(0).getName().equals("External NAT"));
 
     // Tags
-    assert (i.getTags().getItems().size() == NETWORK_TAGS.split(" ").length);
-    assert (i.getTags().getItems().get(0).equals(NETWORK_TAGS.split(" ")[0]));
-    assert (i.getTags().getItems().get(1).equals(NETWORK_TAGS.split(" ")[1]));
+    assertTrue(i.getTags().getItems().size() == NETWORK_TAGS.split(" ").length);
```

Comment:
Not `.equals` but should still be `assertEquals`

Reply:
Done.